crates/services/src/inference_provider_pool/mod.rs

```diff
@@ -473,9 +512,22 @@ impl InferenceProviderPool {
                     .await
                 {
                     Ok(mut attestation) => {
-                        // Remove 'all_attestations' field if present
-                        attestation.remove("all_attestations");
-                        model_attestations.push(attestation);
+                        // Check if the response contains an error field
+                        if attestation.contains_key("error") {
```

Comment:
Just a question, why error attestation data will be returned with 200 status code?

Reply:
Good catch. The error handling of the `get_attestation_report` function is problematic. I have fixed it.